### ISSUE TITLE
Setting useSignificantChangesOnly to true

### ISSUE THREAD

**Issue by gunters63** [transistorsoft/react-native-background-geolocation#67]
This is not a bug, but it is not mentioned in the documentation, at least I couldn't find it.

Testing location tracking in the iOS simulator works fine using the Debug/Location menu.

But as soon as I turn on useSignificantChangesOnly I don't get any location updates anymore at all.
My guess is this is because useSignificantChangesOnly tracks change of cell towers which is not possible in the simulator.

Is this true?

**Comment by christocracy** [comment]
Make sure you disable the `UIBackgroundMode` "location" when booting the app with `useSignificantChangesOnly`.  Also make sure you've configured `LocationAlwaysUsageDescription` in your .plist.


It will operate in the simulator.

**Comment by gunters63** [comment]
NSLocationAlwaysUsageDescription was already set.

Tried with UIBackgroundMode disabled (in Capabilities Background Modes), reinstalled App. But it still didn't work in the simulator.

As we already have an older version of our App which has Background Mode Location Update enabled which got through the review process, I will just use useSignificantChangesOnly false now.

Or is there any other significant advantage (other than being able to use background location off for an easier review), like less power consumption?

**Comment by gunters63** [comment]
Strange thing, in the previous version of out app, UIBackgroundMode was enabled, though useSignificantChangesOnly was set to false (which doesn't make much sense when I understand you correctly?).

Changed useSignificantChangesOnly to true now and uploaded a new version to Testflight.

After the update the app showed a permission screen for "access your motion and fitness activity" once.

When I uninstalled and reinstalled this version only the usual background location update permission dialog showed.
